ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/methods/PrivSubscribeTest.java

```diff
@@ -134,6 +137,31 @@ public void multiTenancyCheckFailure() {
         .hasMessageContaining("msg");
   }
 
+  @Test
+  public void multiTenancyCheckSuccess() {
```

Comment:
is there a test to check multiTenancyCheckFail?

Reply:
As far as I can tell, the previous `multiTenancyCheckFailure` test checks the multi-tenancy fail case.